xil_carma_messenger/README.md

```diff
@@ -0,0 +1,37 @@
+# K900 CARMA Cloud CDASim Integration Scenario
+
+## Introduction
+
+This **CARMA Config** includes the docker-compose and configuration file setup for the **K900 CARMA Cloud CDASim Integration Scenario**.
+
+## Scenario Description
+
+This CARMA Configuration Image creates a **XIL** (Anything-In-the-Loop) scenario which includes **CARLA**, **SUMO** , **NS3** (CV2X Model), a **Virtual Signal Controller**, **CARMA Streets** and **CARMA Platform**. The scenario configured is meant to show base basic CDASim functionality along with the new developed **CARMA Cloud** integration. **CARMA Cloud** is a cloud application used to maintain and provide traffic controls for autonomous vehicles to use to update their local maps. The diagrams below illustrate the configured scenario including vehicle routes and infrastructure location.
+
+
+![Alt text](docs/town04_diagram.png)
+
+![Alt text](docs/scenario_diagram.png)
+
+## Simulators
+
+| Simulator      | Version |
+| ----------- | ----------- |
+| CARLA      | 0.9.10       |
+| SUMO      | 1.15       |
+
+## Deployment Instructions
+
+1) Copy all files in the `cdasim_config/route_config` directory to  directory to `/opt/carma/routes/`
```

Comment:
for example, this should also include copying of routes into the /opt/carma/messenger/messenger_1 (OR ask them to create a symlink as you mentioned before)